test/e2e_test.go

```diff
@@ -138,7 +142,59 @@ func TestE2EPutGetLogicForEigenDAStore(t *testing.T) {
 	require.Equal(t, testPreimage, preimage)
 }
 
-func TestE2EPutGetLogicForMemoryStore(t *testing.T) {
+func TestHoleskyWithProxyClient(t *testing.T) {
+	if !runTestnetIntegrationTests {
+		t.Skip("Skipping testnet integration test")
+	}
+
+	ts, kill := createTestSuite(t, false)
+	defer kill()
+
+	cfg := &client.Config{
+		URL: fmt.Sprintf("%s://%s:%d", transport, host, port),
+	}
+	daClient := client.New(cfg)
+	t.Log("Waiting for client to establish connection with plasma server...")
+	// wait for server to come online after starting
+	wait.For(ts.ctx, time.Second*1, func() (bool, error) {
+		err := daClient.Health()
+		if err != nil {
+			return false, nil
+		}
+
+		return true, nil
+	})
```

Comment:
I believe that createTestSuite() is already waiting for the server to come up.

https://github.com/Layr-Labs/eigenda-proxy/blob/f270ca10fe3c7be238db3e71ef7d367f6454eeb5/server/server.go#L114

Reply:
this just does a time wait and verifies that no error is generated within 10 ms. Feel like doing the await gurantees client <-> server communication is ready versus instantiation the server construct